src/blockdev.rs

```diff
@@ -72,8 +72,10 @@ struct BlkDev {
 
 fn get_partitions(device: &str) -> Result<Vec<BlkDev>> {
     // have lsblk enumerate partitions on the device
+    // Older lsblk, e.g. in CentOS 7.6, doesn't support PATH, but -p option
     let result = Command::new("lsblk")
         .arg("--pairs")
+        .arg("-p")
```

Comment:
Can we use `--paths` here instead to be more clear?

Reply:
agree. done